integrationtests/controller/bundle/suite_test.go

```diff
@@ -57,19 +58,14 @@ var _ = BeforeSuite(func() {
 	var err error
 	cfg, err = testEnv.Start()
 	Expect(err).NotTo(HaveOccurred())
-	Expect(cfg).NotTo(BeNil())
 
 	utilruntime.Must(v1alpha1.AddToScheme(scheme.Scheme))
 	//+kubebuilder:scaffold:scheme
 
 	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
 	Expect(err).NotTo(HaveOccurred())
-	Expect(k8sClient).NotTo(BeNil())
```

Comment:
Why would `client.New` return a nil client and no error? That would be strange. The expectation confused me and I read the code. I can't see any situation where it would return nil for a client and no error.